### PR TITLE
Add undocumented (but very useful) parameters

### PR DESCRIPTION
**Pull request by ErikEJ**
source: https://blogs.msdn.microsoft.com/ssdt/2016/10/20/sql-server-data-tools-16-5-release/

### CODE REVIEW

**Comment by PRMerger15** [comment]
@ErikEJ : Thanks for your contribution! The author, @pensivebrian, has been notified to review your proposed change.

File: docs/tools/sqlpackage.md [hunk 1]
```diff
@@ -96,6 +96,8 @@ A SqlPackage.exe publish operation incrementally updates the schema of a target
 |**/AccessToken:**|**/at**|{string}| Specifies the token based authentication access token to use when connect to the target database. |
 |**/AzureKeyVaultAuthMethod:**|**/akv**|{Interactive&#124;ClientIdSecret}|Specifies what authentication method is used for accessing Azure KeyVault |
 |**/ClientId:**|**/cid**|{string}|Specifies the Client ID to be used in authenticating against Azure KeyVault, when necessary |
+|**/DeployScriptPath:**|**/dsp**|{string}|This is an optional path to write the deployment script to. For Azure deployment, if there were TSQL commands to create of modify the DB a master script will be written to the same path but with “Filename_Master.sql” as the output file name. |
```
Comment (pensivebrian) [inline docs/tools/sqlpackage.md]: How about the following wording, which is more inline with the existing docs: Specifies an optional file path to output the deployment script. For Azure deployments, if there are TSQL commands to create or modify the master database, a script will be written to the same path but with "Filename_Master.sql" as the output file name.

File: docs/tools/sqlpackage.md [hunk 2]
```diff
@@ -96,6 +96,8 @@ A SqlPackage.exe publish operation incrementally updates the schema of a target
...
+|**/DeployReportPath:**|**/drp**|{string}|This is an optional path to write the deployment report xml file to. |
```
Comment (pensivebrian) [inline docs/tools/sqlpackage.md]: How about the following, again to be more inline with existing doc wording: Specifies an optional file path to output the deployment report xml file.

**Comment by PRMerger15** [comment]
@ErikEJ : Thanks for your contribution! The author, @pensivebrian, has been notified to review your proposed change.

File: docs/tools/sqlpackage.md (hunk 2 above)
Comment (ErikEJ) [reply docs/tools/sqlpackage.md]: PR updated

**Comment by pensivebrian** [comment]
@ktoliver - I've reviewed and signed off on this change. Do I have to do anything to get this merged? Add or remove a tag?

**Comment by ktoliver** [comment]
Adding # sign-off (with space after pound sign deleted) lets a PR reviewer know the PR is ready for review and merge.

**Pull request merged**